src/OpenAL/OpenToolkit.OpenAL/Native/ApiContainer.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using OpenToolkit.Core.Loader;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+#pragma warning disable SA1649 // File name should match first type name
+    public abstract class ApiBase
+#pragma warning restore SA1649 // File name should match first type name
+    {
+#pragma warning disable SA1401 // Fields should be private
```

Comment:
Im confused about this one too, nobody but the class is accessing this field. Unless this is a remainder of some former refactoring

Reply:
It's probably a remainder